new/lib/userSessionApi.tsx

```diff
@@ -15,23 +18,27 @@ function recoverToken() {
 }
 
 export async function loginWithUidAndPassword(uid: string, password: string): Promise<string> {
-    // TODO mocking login
     console.log("sending to loginWithUidAndPassword: " + uid + " | " + password);
-    await new Promise((r) => setTimeout(r, 1000));
-    if (uid == "test" && password == "test") {
-        sessionToken = "123-123-123";
+    // await new Promise((r) => setTimeout(r, 1000));
+    const person: UserLogin = {
+        userId: uid,
+        logInPasswordHashed: password,
```

Comment:
will the password be hashed on client-side or server-side?

Reply:
you are right, at this moment its not hashed, but i think we will hash it before we sent it to the server by encryption?